hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -179,6 +179,20 @@ public class HoodieCompactionConfig extends HoodieConfig {
           + "record size estimate compute dynamically based on commit metadata. "
           + " This is critical in computing the insert parallelism and bin-packing inserts into small files.");
 
+  public static final ConfigProperty<String> COPY_ON_WRITE_RECORD_DYNAMIC_SAMPLE_MAXNUM = ConfigProperty
```

Comment:
I see that you're following a pattern there, but i don't think we should prefix COPY_ON_WRITE since there's no reason for us to limit usage of this config to just COW (we can re-use it for similar purposes for MOR as well)

Reply:
got it, i will update this code